### ISSUE TITLE
whitespace in cron file issue

### ISSUE THREAD

**Issue by 0bl1t3r4t3** [rackerlabs/auter#86]
It appears that tabs are not being properly handled in the /etc/cron.d/auter file.  When the cron job tries to run, the command seems to be appending the full line as an option when tabs are used.   This causes the job to fail.  Here is an example of the cron log entries.

Dec 13 11:00:01 staging CROND[24583]: (root) CMD (/usr/bin/auter --prep^I^I^I^I       # Prep Every Wednesday at 11:00)
Dec 13 12:00:01 staging CROND[25482]: (root) CMD (/usr/bin/auter --apply^I^I^I       # Apply Every Wednesday at 12:00 (AUTOREBOOT is set in /etc/auter/auter.conf))

I checked the cache, logs and uptime to confirm that neither the --prep or --apply succeeded.

**Comment by reaperzn** [comment]
Thanks @0bl1t3r4t3 for raising this issue. Can you confirm some details about your setup as well as the exact line that you uncommented from /etc/cron.d/auter. Based on your output above, the issue seems to be related to the ```^|^|^|``` directly after the auter command. You will probably find the details of the problem in your local root mailbox. 

Could you confirm:
1) Do those ```^I``` characters appear in the /etc/cron.d/auter file. 
2) Which OS (CentOS/RHEL/Ubuntu) and what version
3) What version of auter
4) What shell is being configured for your environment
5) How did you edit that cron file (Was it edited in Windows or does your text editor have config to convert a  set number of spaces to tabs) 

Basically it seems that this was likely a text editor that caused this problem, the example cron file should not contain tabs, we have used spaces instead. 

However to avoid this possibility in future, I have submitted a PR which moves the comments to the line above. See PR #88

**Comment by 0bl1t3r4t3** [comment]
The systems that auter is installed on are all customer devices with the same build and configuration.  See the answers to your questions below.  I simply copied lines 28 and 29, creating lines 32 and 33, then edited the day/time. This affected all of the systems that it was installed on.

Answers:
1. No
2. CentOS release 6.9 (Final)
3. auter 0.10
4. /bin/bash 4.1.2(2)-release
5. vim 7.4.629
6. directly with vim and each device

**Comment by reaperzn** [comment]
@0bl1t3r4t3 This has now been addressed and merged. This change will be included in the next release.
